src/util/api.js

```diff
@@ -1,7 +1,17 @@
-import { Projects } from './mock';
+import { Projects, Members } from './mock';
 import { request } from './request';
 
 const ping = () => request('/ping/v1/ping');
 const getProjectsMock = () => Projects;
-
-export const api = { ping, getProjectsMock };
+const getMembersMock = () => Members;
+const changePasswordMock = (/* newPassword */) => {
+  if (Math.random() < 0.5) throw new Error('NO CONNECTION');
+  return { status: 'SUCCESS' };
```

Comment:
Let's use 'result' field instead of 'status' one (to be consistent with ping API).
P.S. If we will really need status, I'd like it to be a number (0 as OK, -1 as error, like process exit code)

Reply:
UPD: I was wrong. It is ok